pkg/cmd/bootstrap.go

```diff
@@ -175,19 +204,23 @@ func initiateInteractiveMode(io *BootstrapParameters, client *utility.Client) er
 		io.GitHostAccessToken = ui.EnterGitHostAccessToken(io.ServiceRepoURL)
 	}
 	io.ServiceWebhookSecret = ui.EnterServiceWebhookSecret()
```

Comment:
I thought we wouldn't ask the user for access token if token already exists in keyring.